src/main/java/io/confluent/connect/jdbc/util/ConnectionProvider.java

```diff
@@ -51,4 +51,14 @@ boolean isConnectionValid(
    * Close this connection provider.
    */
   void close();
+
+  /**
+   * Get the publicly viewable identifier for this connection provider and / or the database.
+   * The resulting value should not contain any secrets or passwords.
+   *
+   * @return the identifier; never null
+   */
+  default String identifier() {
+    return toString();
+  }
```

Comment:
Step 1: In order to log useful information about the connection providers, we add a new `identifier` method that returns a string, and default to the `toString()` method.